src/lib/pages/backoffice/Loan/LoanDetails/AvailableItems/AvailableItems.js

```diff
@@ -64,11 +65,15 @@ export default class AvailableItems extends Component {
 
   checkoutItemButton(item, loan) {
     const { performCheckoutAction } = this.props;
+    const { isLoading } = this.state;
```

Comment:
this should be mapped via redux with ACTION_IS_LOADING of /modules/Loan/actions.js, not with the internal state of the component, otherwise if the checkout fails, the page will not reload and the button will be stuck in loading state